projects/api-twitter/server/utils/email.js

```diff
@@ -0,0 +1,25 @@
+// 1. importar la librería - npm install nodemailer
```

Comment:
porfa los comments en ingles

Reply:
// 1. import the library - npm install nodemailer